salt-blackout-formula/metadata/form.yml

```diff
@@ -0,0 +1,21 @@
+minion_blackout:
+  $type: boolean
+  $default: False
+  $name: Blackout
+  $help: Disable all remote execution commands
+
+minion_blackout_whitelist:  
+  $type: edit-group
+  $name: Blackout whitelist
+  $help: Modules allowed during blackout. Removing the default values is not recommended because it can prevent Uyuni from working properly.
+  $disabled: "!formValues.minion_blackout"
+  $prototype:
+    $type: text
+  $default:
+  - test.ping
+  - grains.item
+  - grains.items
+  - mgractionchains.start
+  - mgractionchains.start
```

Comment:
Why is the `mgractionchains.start` listed twice?

Reply:
Removed `mgractionchains` completely. Not needed in the first iteration.